scripts/dts/edtlib.py

```diff
@@ -906,8 +906,8 @@ def _init_prop(self, prop_spec):
         if not prop_type:
             _err("'{}' in {} lacks 'type'".format(name, self.binding_path))
 
-        val = self._prop_val(name, prop_type, prop_spec.required,
-                             prop_spec.default)
+        val = self._prop_val(name, prop_type, prop_spec.deprecated,
```

Comment:
Is it time to just pass `prop_spec`?

Reply:
This issue is we need a "fake" prop_sec for the other caller of `_prop_val`